cherryontech/unmatched
# Feature/biopage updates

## Description


This PR adds bio page details for Maggie, James, Jedidah, Maya, Rana and Tazin. Please review for typos or any inconsistencies 


## Related Tickets & Documents


## Acceptance Criteria



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|    ✓ | Copy             |


## Mobile & Desktop Screenshots/Recordings
<img width="1292" alt="Screenshot 2023-11-19 at 7 08 24 PM" src="https://github.com/cherryontech/unmatched/assets/110904882/cb56fa46-c6d2-4c79-ac33-f35f0d2454f4">



## What I learned


## Testing steps

Switch to this branch with the following command: git switch feature/biopage-updates
Run the following scripts: npm run dev to start Vite's dev build 
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and then to each individual’s bio pages
On each bio page, read through the copy, check for typos, makes suggestions, and check the links to make sure they work.

## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |



  ![In PROGRESS](https://media.giphy.com/media/VHOF8pfPZOt9p018zw/giphy.gif)
